tests/Transports.AspNetCore.Tests/SystemTextJsonTests.cs

```diff
@@ -41,64 +41,61 @@ public async Task Decodes_BigInteger()
             var request = @"{""variables"":{""a"":1234567890123456789012345678901234567890}}";
             var ret = await Deserialize(request);
             var bi = BigInteger.Parse("1234567890123456789012345678901234567890");
-            ret.Single.Inputs["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
+            ret.Single().Variables["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
         }
 
         [Fact]
         public async Task Dates_Should_Parse_As_Text()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Variables["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
         }
 
         [Fact]
         public async Task Extensions_Null_When_Not_Provided()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Extensions.ShouldBeNull();
+            ret.Single().Extensions.ShouldBeNull();
         }
 
         [Fact]
         public async Task Name_Matching_Is_Case_Sensitive()
         {
-            var ret = await Deserialize(@"{""VARIABLES"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs.ShouldBeNull();
+            var exception = await Should.ThrowAsync<System.Text.Json.JsonException>(()
```

Comment:
The same here. Imagine 
```json
{
   "query": ....
   "VARIABLES": ....   <----- BOOM
   "variables": ....       <----- even not seen
}
```

Reply:
I'm not exactly sure what your point is.

Reply:
Deserializer should just skip unknown properties. My example shows stupid situation when deserializer throws being from one step from the needed property. You changed behavior in main repo, thanks.

Reply:
Well, it was designed to throw an exception no matter the order.  In any case, it is changed now as you suggested.